elementary/monitor/alerts/test_alert.py

```diff
@@ -152,22 +152,28 @@ def data(self) -> Dict:
 
     @property
     def concise_name(self) -> str:
-        if self.is_elementary_test:
+        if self.test_sub_type_display_name.lower() not in (
```

Comment:
`concise_name` is used on the group_by method both in Slack and Teams.
Make sure that this change is not breaking anything 🙏🏻

Reply:
yeah I want to remove this unnecessary thing in the group alerts too